src/pages/components/map-related/MapLayers/BusToolTip.tsx

```diff
@@ -39,7 +41,14 @@ export function BusToolTip({ position, icon }: BusToolTipProps) {
         <>
           <header className="header">
             <h1 className="title">
-              {t('line')} :<span>{siriRide && siriRide.gtfsRouteRouteShortName}</span>
+              {t('line')} :
+              <span>
+                {siriRide ? (
```

Comment:
```suggestion
```
now this check isn't needed

Reply:
I left it like this because the whole file is written in this style, but maybe I'll remove it from the whole file since it's not needed inside the returned JSX. sounds good?

Reply:
sounds great! let's keep things minimal wherever it's possible

Reply:
(feel free to hit the merge button if you want to)